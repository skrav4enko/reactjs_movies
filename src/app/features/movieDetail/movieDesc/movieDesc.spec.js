/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import MovieDesc from '.';

describe('Movie Desc', () => {
  let defaultProps;

  beforeEach(() => {
    defaultProps ={
      movie: {
        id: 338970,
        title: 'Tomb Raider',
        tagline: 'Her legend begins',
        vote_average: 6.2,
        vote_count: 817,
        release_date: '2018-03-08',
        poster_path: 'https://image.tmdb.org/t/p/w500/ePyN2nX9t8SOl70eRW47Q29zUFO.jpg',
        overview:
          'Lara Croft, the fiercely independent daughter of a missing adventurer.',
        budget: 94000000,
        revenue: 126025000,
        genres: ['Action', 'Adventure'],
        runtime: 118,
      }
    }
  });

  it('should match snapshot', () => {
    const wrapper = shallow(<MovieDesc {...defaultProps} />);
    expect(wrapper).toMatchSnapshot();
  });
});

