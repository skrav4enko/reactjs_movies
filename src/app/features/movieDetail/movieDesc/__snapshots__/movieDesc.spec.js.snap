// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Desc should match snapshot 1`] = `
<div
  className="movie-desc"
>
  <div
    className="container"
  >
    <div
      className="movie-desc__poster"
    >
      <img
        alt="Tomb Raider"
        className="movie-desc__img"
        src="https://image.tmdb.org/t/p/w500/ePyN2nX9t8SOl70eRW47Q29zUFO.jpg"
      />
    </div>
    <div
      className="movie-desc__content"
    >
      <div
        className="movie-desc__header"
      >
        <h2
          className="movie-desc__title"
        >
          Tomb Raider
        </h2>
        <Rating
          value={6.2}
        />
      </div>
      <div
        className="movie-desc__genres"
      >
        Action, Adventure
      </div>
      <div
        className="movie-desc__params"
      >
        <Params
          dimension="year"
          value="2018"
        />
        <Params
          dimension="min"
          value={118}
        />
      </div>
      <p
        className="movie-desc__overview"
      >
        Lara Croft, the fiercely independent daughter of a missing adventurer.
      </p>
    </div>
  </div>
</div>
`;
